Microsoft.Toolkit.Uwp.Services/Exceptions/RequestFailedException.cs

```diff
@@ -44,13 +43,23 @@ public RequestFailedException(string message)
         /// </summary>
         /// <param name="statusCode">Failure status code.</param>
         /// <param name="reason">Failure reason.</param>
-        public RequestFailedException(HttpStatusCode statusCode, string reason)
+        public RequestFailedException(Windows.Web.Http.HttpStatusCode statusCode, string reason)
```

Comment:
Agreed, But talking about using that when we have to throw exception like we have to use our own `RequestFailedException` here.